README.md

```diff
@@ -12,12 +12,47 @@ This method is considered advanced and should only be used if one is an expert i
 
 ## Installation
 
-Run as root (sudo su):
+Section 1 provides links to download Debian 10 and information on installation. If you already have Debian 10 installed and wish to move on to installing Home Assistant, proceed to Section 2. 
+
+### Section 1 – Install Debian 10
+
+**1.1)** Start by downloading `debian-live-10.5.0-amd64-standard.iso.torrent` from [HERE](https://cdimage.debian.org/debian-cd/current-live/amd64/bt-hybrid/). This is a torrent file, so use your favourite torrent program. Alternitively, you can download the `debian-live-10.5.0-amd64-standard.iso` ISO from [HERE](https://cdimage.debian.org/debian-cd/current-live/amd64/iso-hybrid/).
+
+Some useful infomation on how to install Debian 10 can be found on the Debain website [HERE](https://www.debian.org/releases/stable/installmanual)
+
+
+### Section 2 – Install Home Assistant Supervised
+
+With Debian 10 installed, you can move on to installing Home Assistant Supervised.
+
+**2.1)** Start by updating Debian to ensure all the latest updates and security patches are installed. To do this, log into the terminal of your machine, enter the following command.
+
+```
+sudo apt update && sudo apt dist-upgrade -y && sudo apt autoremove -y
+```
+
+Depending on the speed of your internet connection, this could take anywhere from 30 seconds to 20 minutes to complete. When finished, you will see the prompt.
+
+**2.2)** With the operating system up to date, you can install Home Assistant Supervised. Enter each line of the below commands into the terminal and execute them one at a time.
```

Comment:
"operating system up to date" => there is no such requirement. The requirements are listed in the ADR, an updated system does not have to match the requirements.

Reply:
If I recall, it's been a while, I thought the deb installer ran updates at the end of the install anyways?
Might just have been the preseed I was using 🤷‍♂️